src/server/routers/scope.ts

```diff
@@ -133,6 +133,44 @@ export const scopeRouter = t.router({
 
       return scope;
     }),
+
+  createMany: t.procedure
```

Comment:
I think we can setup a specific route for the import of suggested scopes and issues that will manage both instead of manage separately on client side.

We can imagine a `createFromSuggested` route